src/hrflow_connectors/connectors/hrflow/warehouse/profile.py

```diff
@@ -248,6 +254,20 @@ def write_parsing(
         elif source_response["data"]["sync_parsing"] is True:
             current_profile = parsing_response["data"]["profile"]
             profile_result = merge_info(current_profile, profile_info)
+
+            profile_result["text"] = profile_text or profile_result.get("text", "")
+            profile_result["text_language"] = (
+                profile_text_language or profile_result.get("text_language", "")
+            )
+            profile_result["experiences_duration"] = (
+                profile_experiences_duration
+                or profile_result.get("experiences_duration", 0)
+            )
```

Comment:
Si dans le `profile` de base tu as `experience_duration = 0` c'est quoi le résultat voulu ? 

Parce que ici ce qui se passe c'est que 0 va evaluer à `False` et tu coup tu vas passer à ce qu'il y'a dans `profile_result`. 

Est ce que c'est bien ca que tu veux ?

Reply:
L'idée c'est de faire fois à tous ce qu'est dans le profile json. Est-ce qu'on doit faire:
`profile_experiences_duration=profile.get("experiences_duration")
profile_result["experiences_duration"] = profile_experiences_duration if not None else profile_result.get("experiences_duration")
`

Reply:
Ce que je comprends de ce que tu dis c'est que dés que la clé est présente dans `profile` et differente de `None` tu veux prendre ca forcement. 

Dans ce cas tu fais 
```python
profile_result["X"] = profile.get("X") if profile.get("X") is not None else profile_result.get("X")
```

Reply:
OK parfati! par contre pour les textes s'ils sont vides on ne va prendre que ce qui est parsé

Reply:
@the-forest-tree I did modify the code thanks